bedrock/settings/base.py

```diff
@@ -24,29 +26,31 @@ def path(*args):
 
 
 # Is this a dev instance?
-DEV = False
+DEV = config('DEV', cast=bool, default=False)
 
-DEBUG = False
-TEMPLATE_DEBUG = DEBUG
+DEBUG = TEMPLATE_DEBUG = config('DEBUG', cast=bool, default=False)
 
 # Production uses MySQL, but Sqlite should be sufficient for local development.
 # Our CI server tests against MySQL.
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'bedrock.db',
-    }
-}
-SLAVE_DATABASES = []
+DATABASES = config(
```

Comment:
This is a bit hard to parse as a human, but it seems like it'll work well as a human managing the environment.